batch/batches/Concat/KAsyncConcat.class.php

```diff
@@ -357,8 +393,8 @@ protected function concatFiles($ffmpegBin, $ffprobeBin, array $filesArr, $outFil
 			{
 				$cmdStr .= "-protocol_whitelist \"concat,file,subfile,https,http,tls,tcp,file\" ";
 			}
-			
-			$cmdStr .= "$probeSizeAndAnalyzeDurationStr -i $concatStr $videoParamStr $audioParamStr";
+
+			$cmdStr .= "$probeSizeAndAnalyzeDurationStr $concatStr $videoParamStr $audioParamStr";
```

Comment:
we add the -i when constructing the $concatStr, 
multi source concat: since we have multiple input files and we use the filter complex to concat, we have multiple -i in the concatStr
for reference https://trac.ffmpeg.org/wiki/Null